state/staking/types.ts

```diff
@@ -0,0 +1,31 @@
+import type { EscrowData } from 'sdk/services/kwentaToken';
+import { ClaimParams } from 'sdk/services/kwentaToken';
+import { FetchStatus } from 'state/types';
+
+export type StakingState = {
+	kwentaBalance: string;
+	vKwentaBalance: string;
+	veKwentaBalance: string;
+	escrowedKwentaBalance: string;
+	claimableBalance: string;
+	totalStakedBalance: string;
+	stakedEscrowedKwentaBalance: string;
+	stakedKwentaBalance: string;
+	epochPeriod: number;
+	weekCounter: number;
+	kwentaAllowance: string;
+	vKwentaAllowance: string;
+	veKwentaAllowance: string;
+	totalVestable: number;
```

Comment:
I think we should avoid using number anywhere for amounts as we use Wei everywhere else throughout the app, so best to remain consistent.

Reply:
Will fix